client/src/components/main/coffee_shop/CoffeeShopIndex.scss

```diff
@@ -1,52 +1,31 @@
 .panel-coffee-subtitle {
   font-weight: 600;
-  font-size: 13px;
   color: #515151;
   margin-bottom: 10px;
 }
 
 .panel-coffee {
-  font-size: 14px;
   margin-top: 15px;
 }
 
-.panel-processing {
-  svg {
-    fill: brown;
-    margin-right: 5px;
-  }
-}
-
-.panel-origin {
-  svg {
-    fill: green;
-  }
-}
-
-.panel-flavors {
-  svg {
-    fill: #805000;
-  }
-}
-
 .panel-processing, .panel-origin, .panel-flavors {
   display: flex;
   align-items: center;
-  line-height: 1;
-  svg {
```

Comment:
Does removing the height and width not create any sizing issues? Might not but I for some reason thought it did.

Reply:
The line height was actually being declared for this in a few different places, it seems that multiple parts were setting it to either 1 or 1.1. I think it ended up being set to 1.1 after removing this. I increased the size of the icons in the index to 25px by 25px to match the search icon and everything was made a little bigger in the index as a result. So a line height of 1.1 ended up resulting in a pretty similar appearance to what it was like before imo.